src/components/baseComponent.ts

```diff
@@ -78,6 +79,23 @@ export abstract class MgtBaseComponent extends LitElement {
 
   private static _useShadowRoot: boolean = true;
 
+  /**
+   * A flag to check if the component's firstUpdated method has fired.
+   *
+   * @protected
+   * @memberof MgtBaseComponent
+   */
+  protected get loading(): boolean {
```

Comment:
Same as the methods, does it make sense to call the boolean `isLoadingState`?

Reply:
Good idea. I like the clarity